internal/installcfg/installcfg.go

```diff
@@ -210,11 +223,28 @@ func setBMPlatformInstallconfig(log logrus.FieldLogger, cluster *common.Cluster,
 	return nil
 }
 
+func applyConfigOverrides(overrides string, cfg *InstallerConfigBaremetal) error {
```

Comment:
all the override parameters are added in the end? there is nothing that will replace existing config?

Reply:
I'm confused. I'm unpacking the overrides on top of the existing config. So if you provide the same key in the overrides it will .. well .. override :laughing: 

There are tests that show this behavior.